apps/etterlatte-saksbehandling-ui/client/src/components/person/aktivitet/utils.ts

```diff
@@ -0,0 +1,70 @@
+import {
+  AktivitetspliktUnntakType,
+  AktivitetspliktVurderingType,
+  IAktivitetspliktAktivitetsgrad,
+  IAktivitetspliktUnntak,
+  IAktivitetspliktVurderingNy,
+} from '~shared/types/Aktivitetsplikt'
+
+export enum AktivitetspliktStatus {
+  OPPFYLT,
+  IKKE_OPPFYLT,
+  UNNTAK,
+  VARIG_UNNTAK,
+  IKKE_VURDERT,
+}
+
+interface StatusPaaAktivitetsplikt {
+  aktivitetspliktStatus: AktivitetspliktStatus
+  dato?: Date | string
+}
+
+const sisteGjeldendeUnntak = (unntak: IAktivitetspliktUnntak[]): IAktivitetspliktUnntak => {
+  return unntak.reduce((a, b) => {
+    return new Date(a.fom) > new Date(b.fom) ? a : b
+  })
+}
+
+const sisteGjeldendeAktivitet = (aktivitet: IAktivitetspliktAktivitetsgrad[]): IAktivitetspliktAktivitetsgrad => {
+  return aktivitet.reduce((a, b) => {
+    return new Date(a.fom) > new Date(b.fom) ? a : b
+  })
+}
+
+export const finnStatusPaaAktivitetsplikt = (
+  aktivitetspliktVurdering: IAktivitetspliktVurderingNy
+): StatusPaaAktivitetsplikt => {
+  // Hvis innbygger har unntak, så viker dette for aktivitetsplikt
+  if (!!aktivitetspliktVurdering.unntak?.length) {
```

Comment:
Unntak slettes ikke etter det er utgått, så det er mulig man her kan ende opp med å vise frem unntak når man egentlig skal vise frem aktivitetsgrad. Det vil da være greit å sjekke om det er unntaket eller aktivitetsplikt som gjelder for dagens dato når man setter status.